pkg/runtime/wfengine/wfengine.go

```diff
@@ -49,15 +49,32 @@ type WorkflowEngine struct {
 }
 
 var (
+	WorkflowActorType   = actors.InternalActorTypePrefix + "wfengine.workflow"
+	ActivityActorType   = actors.InternalActorTypePrefix + "wfengine.activity"
 	wfLogger            = logger.NewLogger("dapr.runtime.wfengine")
 	errExecutionAborted = errors.New("execution aborted")
 )
 
+// WFConfig is the configuration for the workflow engine
+type WFConfig struct {
+	AppID string
+}
+
+func getNamespace() string {
+	namespace := os.Getenv("NAMESPACE")
```

Comment:
I tried to look for ways to fetch namespace, and it seems we are doing similar at some other places too in runtime: https://github.com/dapr/dapr/blob/e988ea9b92f715a59aa3e8bacba8863b97ef4ce6/pkg/runtime/runtime.go#L356

Reply:
Instead of copying that logic, let's instead make a call into the existing API to referenced to get the namespace. That way if they update it to support more than just environment variables, our logic will continue to be consistent with the rest of the system.

Reply:
Found one place in sentry, where they had similar use of using `default`, if nothing found.
I have placed a common logic in utils.